docs/guides/development/javascript/authenticating-over-websockets-jwt/index.md

```diff
@@ -2,8 +2,8 @@
 slug: authenticating-over-websockets-with-jwt
 author:
   name: Nathaniel Stickman
-description: "WebSockets enable real-time communications between clients and server and they can be immensely useful in a variety of web applications. JSON Web Tokens can provide lightweight and secure authentication. They pair well with WebSockets to ensure connections are authorized. In this guide, you learn more about both WebSockets and JSON Web Tokens in this guide. You can also follow an example showing you how to use them together in your application."
-og_description: "WebSockets enable real-time communications between clients and server and they can be immensely useful in a variety of web applications. JSON Web Tokens can provide lightweight and secure authentication. They pair well with WebSockets to ensure connections are authorized. In this guide, you learn more about both WebSockets and JSON Web Tokens in this guide. You can also follow an example showing you how to use them together in your application."
+description: "In this guide, you learn more about both WebSockets and JSON Web Tokens will also follow an example showing you how to use them together in your app."
```

Comment:
This sentence construction is not correct. You could write something like "...JSON Web Tokens. Follow an example..."